lib/kernel/fmax_common.cl

```diff
@@ -0,0 +1,26 @@
+/* OpenCL built-in library: fmax_common()
+
+   Copyright (c) 2023 Ben Ashbaugh <ben.ashbaugh@intel.com>
+   
+   Permission is hereby granted, free of charge, to any person obtaining a copy
+   of this software and associated documentation files (the "Software"), to deal
+   in the Software without restriction, including without limitation the rights
+   to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+   copies of the Software, and to permit persons to whom the Software is
+   furnished to do so, subject to the following conditions:
+   
+   The above copyright notice and this permission notice shall be included in
+   all copies or substantial portions of the Software.
+   
+   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+   AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+   THE SOFTWARE.
+*/
+
+#include "templates.h"
+
+DEFINE_EXPR_F_FF(_cl_fmax_common, a < b ? b : a)
```

Comment:
I had to add a prefix to this function or I got an error about duplicate symbols.  Is there a better way to do this?  Should I add these functions to `_builtin_renames.h` for example?

Reply:
Sorry, i forgot about the renames. Yes please, add the functions to `_builtin_renames.h` instead of using the `_cl_` prefix directly.

Reply:
Thanks - done!